src/script/components/MessagesList/Message/MemberMessage/MessageContent.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+ * Wire
+ * Copyright (C) 2023 Wire Swiss GmbH
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://www.gnu.org/licenses/.
+ *
+ */
+
+import {MemberMessage as MemberMessageEntity} from 'src/script/entity/message/MemberMessage';
+import {useKoSubscribableChildren} from 'Util/ComponentUtil';
+
+export function MessageContent({message}: {message: MemberMessageEntity}) {
+  const {htmlCaption} = useKoSubscribableChildren(message, ['htmlCaption']);
+  return <p className="message-header-caption" dangerouslySetInnerHTML={{__html: htmlCaption}}></p>;
+}
```

Comment:
In a follow up PR, this will be where the rendering will be done (no more `dangerouslySetInnerHTML` in there)